conformance/packages/dns-test/src/tshark.rs

```diff
@@ -24,22 +26,28 @@ impl Container {
 
         let tshark = format!(
             "echo $$ > {pidfile}
-exec tshark --log-level debug --log-domain main,capture -l -i eth0 -T json -O dns -f 'udp port {UDP_PORT}'"
+exec tshark -l -i eth0 -T json -O dns -f 'udp port {UDP_PORT}'"
         );
         let mut child = self.spawn(&["sh", "-c", &tshark])?;
 
         let stderr = child.stderr()?;
         let mut stderr = BufReader::new(stderr).lines();
 
-        // Read from stdout on another thread. This ensures the subprocess won't block when writing
-        // to stdout due to full pipe buffers, and means we can read from stderr in
-        // `wait_for_capture()`, `wait_for_new_packets()`, or `terminate()` at our own pace without
-        // causing a deadlock.
-        let mut stdout = child.stdout()?;
-        let stdout_handle = thread::spawn(move || -> CoreResult<String, io::Error> {
-            let mut buf = String::new();
-            stdout.read_to_string(&mut buf)?;
-            Ok(buf)
+        let captures_pair = Arc::new((Mutex::new(Vec::<Capture>::new()), Condvar::new()));
```

Comment:
Can you add a little context on what this is trying to do/why we need both a `Mutex` and a `Condvar` here?

Reply:
I added documentation to the fields on `Tshark`